app/views/booking_requests/step_two.html.erb

```diff
@@ -157,51 +157,52 @@
-         </div>
-       </div>
-     </div>
-
-     <div class="SlotPicker-choice is-chosen">
-       <div class="SlotPicker-choiceInner">
-         <div class="SlotPicker-position"><span>2</span></div>
-         <div class="SlotPicker-choiceContent">
-           <p class="SlotPicker-date"><%= slot_as_formatted_date(@booking_request.secondary_slot) %></p>
-           <p class="SlotPicker-time">
-           <%= slot_period(@booking_request.secondary_slot) %>, <%= slot_duration %></p>
-         </div>
-       </div>
-     </div>
-
-     <div class="SlotPicker-choice is-chosen">
-       <div class="SlotPicker-choiceInner">
-         <div class="SlotPicker-position"><span>3</span></div>
-         <div class="SlotPicker-choiceContent">
-           <p class="SlotPicker-date"><%= slot_as_formatted_date(@booking_request.tertiary_slot) %></p>
-           <p class="SlotPicker-time">
-           <%= slot_period(@booking_request.tertiary_slot) %>, <%= slot_duration %></p>
-         </div>
-       </div>
-     </div>
-  </div>
+      <div class="SlotPicker-choice is-chosen">
+        <div class="SlotPicker-choiceInner">
+          <div class="SlotPicker-position"><span>1</span></div>
+          <div class="SlotPicker-choiceContent">
+            <p class="SlotPicker-date"><%= slot_as_formatted_date(@booking_request.primary_slot) %></p>
+            <p class="SlotPicker-time">
+              <%= slot_period(@booking_request.primary_slot) %>, <%= slot_duration %></p>
+          </div>
+        </div>
+      </div>
+
+      <div class="SlotPicker-choice is-chosen">
+        <div class="SlotPicker-choiceInner">
+          <div class="SlotPicker-position"><span>2</span></div>
+          <div class="SlotPicker-choiceContent">
+            <p class="SlotPicker-date"><%= slot_as_formatted_date(@booking_request.secondary_slot) %></p>
+            <p class="SlotPicker-time">
+              <%= slot_period(@booking_request.secondary_slot) %>, <%= slot_duration %></p>
+          </div>
+        </div>
+      </div>
 
-  <p><%= link_to 'Change dates',
+      <div class="SlotPicker-choice is-chosen">
+        <div class="SlotPicker-choiceInner">
+          <div class="SlotPicker-position"><span>3</span></div>
+          <div class="SlotPicker-choiceContent">
+            <p class="SlotPicker-date"><%= slot_as_formatted_date(@booking_request.tertiary_slot) %></p>
+            <p class="SlotPicker-time">
+              <%= slot_period(@booking_request.tertiary_slot) %>, <%= slot_duration %></p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <p><%= link_to 'Change dates',
         booking_request_step_one_location_path(
-          id: location_id,
-          booking_request: {
-            primary_slot: @booking_request.primary_slot,
-            secondary_slot: @booking_request.secondary_slot,
-            tertiary_slot: @booking_request.tertiary_slot
-          }
+            id: location_id,
```

Comment:
This is fixed now.. and think I have corrected the formatting rule that went wrong..